packages/dynamite/dynamite/lib/src/openapi_builder.dart

```diff
@@ -38,6 +39,21 @@ class OpenAPIBuilder implements Builder {
 
   @override
   Future<void> build(final BuildStep buildStep) async {
+    final result = await helperVersionCheck(buildStep);
+
+    try {
+      if (result.messages.isNotEmpty) {
+        if (result.hasFatal) {
+          log.severe(result.messages);
+          return;
+        } else {
+          log.info(result.messages);
+        }
+      }
+    } on Exception catch (exception) {
+      log.warning(exception);
+    }
```

Comment:
I don't get why you'd want to catch these. If there is an exception thrown then the builder should fail, no?

Reply:
The exception is only thrown when we can't read the pubspec in which case we just skip the verification and assume the deps are ok.

Reply:
Then please just log the warning there instead of throwing an exception and handling it here.